modules/cwireBidAdapter.md

```diff
@@ -35,9 +39,12 @@ var adUnits = [
       params: {
         pageId: 1422,                 // required - number
         placementId: 2211521,         // required - number
-        adUnitElementId: 'other_div', // optional, div id to write to, if not set it will default to ad unit code  
+        adUnitElementId: 'other_div', // optional, div id to write to, if not set it will default to ad unit code
```

Comment:
This does not actually do what it says, right? It looks like it's sent over to the backend but has no effect otherwise.

Reply:
This was just some old thing for testing, where we would write in some other div. It is ok that it's only sent to the backend, since then our bootstrap script knows where to write content.

But since this is confusing, I will remove any notion of this param, util someone actually requests this feature.